Marlin/Marlin_main.cpp

```diff
@@ -7551,39 +7552,17 @@ void prepare_move() {
       }
 
       // Update arc_target location
-      arc_target[X_AXIS] = center_X + r_X;
-      arc_target[Y_AXIS] = center_Y + r_Y;
-      arc_target[Z_AXIS] += linear_per_segment;
-      arc_target[E_AXIS] += extruder_per_segment;
+      destination[X_AXIS] = center_X + r_X;
+      destination[Y_AXIS] = center_Y + r_Y;
+      destination[Z_AXIS] += linear_per_segment;
+      destination[E_AXIS] += extruder_per_segment;
 
-      clamp_to_software_endstops(arc_target);
-
```

Comment:
Mainly i wanted to get rid of this call to `clamp_to_software_endstops()` with a parameter different from `destination`.